gsa/src/web/pages/policies/__tests__/detailspage.js

```diff
@@ -234,132 +85,113 @@ beforeEach(() => {
     expect(links[0]).toHaveAttribute(
       'href',
       'test/en/compliance-and-special-scans.html#configuring-and-managing-policies',
     );
 
     expect(links[1]).toHaveAttribute('href', '/policies');
-    expect(icons[1]).toHaveAttribute('title', 'Policies List');
+    expect(screen.getAllByTitle('Policies List')[0]).toBeInTheDocument();
 
-    expect(element).toHaveTextContent('12345');
-    expect(element).toHaveTextContent('Tue, Jul 16, 2019 8:31 AM CEST');
-    expect(element).toHaveTextContent('Tue, Jul 16, 2019 8:44 AM CEST');
-    expect(element).toHaveTextContent('admin');
+    expect(baseElement).toHaveTextContent('234');
+    expect(baseElement).toHaveTextContent('Mon, Aug 17, 2020 2:18 PM CEST');
+    expect(baseElement).toHaveTextContent('Tue, Sep 29, 2020 2:16 PM CEST');
+    expect(baseElement).toHaveTextContent('admin');
 
-    expect(element).toHaveTextContent('bar');
-    expect(element).toHaveTextContent('audit1');
-    expect(element).toHaveTextContent('audit2');
-    expect(element).not.toHaveTextContent('scanner');
+    expect(baseElement).toHaveTextContent('some policy description');
+    expect(baseElement).toHaveTextContent('foo');
+    expect(baseElement).not.toHaveTextContent('scanner');
 
     const detailslinks = getAllByTestId('details-link');
 
-    expect(detailslinks[0]).toHaveAttribute('href', '/audit/1234');
-    expect(detailslinks[1]).toHaveAttribute('href', '/audit/5678');
+    expect(detailslinks[0]).toHaveAttribute('href', '/audit/457');
   });
 
-  test.skip('should render nvt families tab', () => {
-    // NVTFamilies has been refactored so this test won't work until policies are implemented
-    const getPolicy = jest.fn().mockResolvedValue({
-      data: policy,
-    });
-
+  test('should render nvt families tab', async () => {
     const gmp = {
-      [entityType]: {
-        get: getPolicy,
-      },
-      permissions: {
-        get: getPermissions,
-      },
       settings: {manualUrl, reloadInterval},
       user: {
         currentSettings,
         renewSession,
       },
     };
 
+    const [mock, resultFunc] = createGetPolicyQueryMock();
+
+    const [renewSessionQueryMock] = createRenewSessionQueryMock();
+    const [permissionQueryMock] = createGetPermissionsQueryMock({
+      filterString: 'resource_uuid=234 first=1 rows=-1',
+    });
     const {render, store} = rendererWith({
       capabilities: caps,
       gmp,
       router: true,
       store: true,
+      queryMocks: [mock, renewSessionQueryMock, permissionQueryMock],
     });
 
     store.dispatch(setTimezone('CET'));
-    store.dispatch(setUsername('admin'));
 
-    store.dispatch(entityLoadingActions.success('12345', policy));
+    const {baseElement} = render(<Detailspage id="234" />);
 
-    const {baseElement, element, getAllByTestId} = render(
-      <Detailspage id="12345" />,
-    );
+    await wait();
+
+    expect(resultFunc).toHaveBeenCalled();
 
     const spans = baseElement.querySelectorAll('span');
```

Comment:
We should probably use `const tabs = screen.getAllByTestId('entities-tab-title');` now to access the different tabs.